### ISSUE TITLE
vcrun2015 doesn't (re-)install libraries if install is attempted after uninstalling from a prefix

### ISSUE THREAD

**Issue by krnekit** [Winetricks/winetricks#648]
I tried to install vcrun2015, but I got the following message:

> vcrun2015 install completed, but installed file /home/nekit/.wine/dosdevices/c:/windows/system32/mfc140.dll not found

Also, there are other libraries, like MSVCP140.dll that absent.

Wine Staging 1.8, latest winetricks.

**Comment by qwertychouskie** [comment]
32 or 64 bit prefix?

**Comment by krnekit** [comment]
32 bit prefix, configured as Windows XP.
Mageia Linux Cauldron x86_64.

**Comment by austin987** [comment]
Can you please paste/attach a log doing:
WINEPREFIX=~/.winetmptest winetricks -q -v vcrun2015

Make sure you use a clean WINEPREFIX for this.

**Comment by krnekit** [comment]
Logs with WINEARCH=win32 and without.

[winetricks_32.tar.gz](https://github.com/Winetricks/winetricks/files/306005/winetricks_32.tar.gz)
[winetricks_64.tar.gz](https://github.com/Winetricks/winetricks/files/306006/winetricks_64.tar.gz)

**Comment by austin987** [comment]
FYI, if you're going to make a tarball, put both files in it. There's no reason to make a tar for a single file (gzip works for .txt too, or really any format, but compression effectiveness will vary).

**Comment by austin987** [comment]
Both of those logs succeeded, I suspect something is wrong with your original prefix.

**Comment by krnekit** [comment]
> FYI, if you're going to make a tarball, put both files in it. There's no reason to make a tar for a single file (gzip works for .txt too, or really any format, but compression effectiveness will vary).

Ok, I'll do this way next time.

> Both of those logs succeeded, I suspect something is wrong with your original prefix.

Yes. But I figured out how to reproduce the problem. After vcrun installation I uninstalled it using `wine uninstall` and tried to install it again with your command. And it failed with the same error. Log in the attachment, I checked only with `WINEARCH=win32`, but can also check with 64-bit prefix, if needed.
[winetricks_32.tar.gz](https://github.com/Winetricks/winetricks/files/309125/winetricks_32.tar.gz)
